File: sdk-node/examples/contacts/delete.ts
/**
 * Permanently remove a contact.
 *
 * Run:
 *   npx tsx examples/contacts/delete.ts
 */

import { ThreeCommon } from '@3common/sdk'

const client = new ThreeCommon({
  apiKey: '3co_your_api_key_here',
})

const { id } = await client.contacts.delete('cnt_replace_with_real_id')

console.log(`deleted ${id}`)

File: sdk-python/examples/contacts/delete.py
"""Run with: python examples/contacts/delete.py"""

from __future__ import annotations

from threecommon import ThreeCommon


def main() -> None:
    with ThreeCommon(api_key="3co_your_api_key_here") as client:
        result = client.contacts.delete("cnt_replace_with_real_id")
        print(f"deleted {result.id}")


if __name__ == "__main__":
    main()
